config/locales/en.yml

```diff
@@ -1031,6 +1031,18 @@ en:
           tab-title: &payments_h_branding "Payments: H-Branding"
           title: *payments_h_branding
 
+    user_profile:
```

Comment:
per Rails convention, this should go under the action.  so these should go under `users:` (`update:` already exsits there, so the entries should just be added to that section)

(Same with the `sv.yml` entries)

Reply:
I don't think the translations can be found that way.  I'm using the lazy lookup for controllers and views: https://guides.rubyonrails.org/i18n.html#lazy-lookup

Reply:
The controller is `user_profile`, and the actions are listed under that.

Reply:
Hm.  I've been using lazy lookup too.  If it works with how you have them organized then you way is right.  (And that teaches me something more about the organization.)